### PR TITLE
CloneSpy.CloneSpy version 3.43

### PR DESCRIPTION
**Pull request by richardsondev**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16159)

### CODE REVIEW

File: manifests/m/MarcusKleinehagenbrock/CloneSpy/3.43/MarcusKleinehagenbrock.CloneSpy.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: MarcusKleinehagenbrock.CloneSpy
+PackageVersion: 3.43
+PackageLocale: en-US
+Publisher: Marcus Kleinehagenbrock
```
Comment (ItzLevvie) [inline manifests/m/MarcusKleinehagenbrock/CloneSpy/3.43/MarcusKleinehagenbrock.CloneSpy.locale.en-US.yaml:6]:
```suggestion
Publisher: The CloneSpy Team
```
image

File: manifests/m/MarcusKleinehagenbrock/CloneSpy/3.43/MarcusKleinehagenbrock.CloneSpy.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: MarcusKleinehagenbrock.CloneSpy
+PackageVersion: 3.43
+Installers:
+- Architecture: neutral
```
Comment (ItzLevvie) [inline manifests/m/MarcusKleinehagenbrock/CloneSpy/3.43/MarcusKleinehagenbrock.CloneSpy.installer.yaml:6]:
```suggestion
- Architecture: x86
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Other than the changes I've made, everything else looks good to me:
image

File: manifests/m/MarcusKleinehagenbrock/CloneSpy/3.43/MarcusKleinehagenbrock.CloneSpy.locale.en-US.yaml (hunk 1 above)
Comment (richardsondev) [reply manifests/m/MarcusKleinehagenbrock/CloneSpy/3.43/MarcusKleinehagenbrock.CloneSpy.locale.en-US.yaml]: Thanks for your help!

Comment (richardsondev) [reply manifests/m/MarcusKleinehagenbrock/CloneSpy/3.43/MarcusKleinehagenbrock.CloneSpy.locale.en-US.yaml]: I also updated the package identifier with this info

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**